miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -30,11 +30,6 @@
 				Export</a></li>
 	</ul>
 
-	<h2>Sequencing</h2>
-	<ul class="bullets">
-		<li><a href="<c:url value="/miso/pools/ready"/>">Ready to Run</a></li>
```

Comment:
I agree. I thought about this for a while when I first did it, but it came down to ready-to-run being kind of useless as it is. I'll recondier.